### DISCUSSION TITLE
3D buildings implementation?

### DISCUSSION THREAD

**Discussion by mmz15** [uber/react-map-gl#1344] [Q&A] [answered]
Not sure if this is the right place to ask, but is it possible to implement a 3d extrusion of buildings with this Mapbox wrapper? I'm referring to this example specifically: https://docs.mapbox.com/mapbox-gl-js/example/3d-buildings/

**Comment by kylebarron** [comment, marked as answer]
All that example is is a standard Mapbox layer with `'type': 'fill-extrusion'`, so yes.

**Reply by mmz15** [reply]
You're 100% right. Used { Layer } to add a new building layer. Many thanks for pointing that out
